tests/FSharp.Analyzers.Tests/FSharp.Analyzers.Tests.fsproj

```diff
@@ -9,6 +9,7 @@
     </PropertyGroup>
     <ItemGroup>
         <Compile Include="Common.fs"/>
+        <Compile Include="data\disposedBeforeAsyncRun\LetBeforeUseThenMatch.fs" />
```

Comment:
Convention is to just let this be `None Include`d as in line 24 - I presume this was for debugging (or else it was my mistake that you've inherited)

Reply:
yep, I didn't notice Rider doing that.